ports/libfido2/vcpkg.json

```
@@ -0,0 +1,13 @@
+{
+  "name": "libfido2",
+  "version": "1.7.0",
+  "description": "Provides library functionality to communicate with a FIDO device over USB, and to verify attestation and assertion signatures.",
+  "homepage": "https://developers.yubico.com/libfido2/",
+  "supports": "!uwp & (x86 | x64) & windows",
+  "dependencies": [
+    "libcbor",
+    "openssl",
+    "vcpkg-cmake",
```

```suggestion
   {
     "name": "vcpkg-cmake",
     "host": true
   }
```
We should add the dependency like this. Please use `./vcpkg format-manifest ./ports/libfido2/vcpkg.json` to format this file.